openedx/core/lib/xblock_utils.py

```diff
@@ -302,7 +302,9 @@ def add_staff_markup(user, has_instructor_access, disable_staff_debug_info, bloc
         # check that the course is a mongo backed Studio course before doing work
         is_studio_course = block.course_edit_method == "Studio"
 
-        if is_studio_course:
+        is_ccx_course = hasattr(course_id, 'ccx')
+        # preventing users from opening ccx courses in Studio, since Studio doesn't support CCX courses
+        if is_studio_course and not is_ccx_course:
             # build edit link to unit in CMS. Can't use reverse here as lms doesn't load cms's urls.py
             edit_link = "//" + settings.CMS_BASE + '/container/' + unicode(block.location)
```

Comment:
Hi @ormsbee, it has been hard to me understand where (and how) I should do the change. Looking at the `/djangoapps/ccx/overrides.py:get_override_for_ccx` method, I just tried to check the content of `block.course_edit_method`, but I got a recursive loop error.  Here is the log: http://pastebin.com/P6WH1mLi

I searched for `course_edit_method` in the project and I found this:
```python
course_edit_method = String(
        display_name=_("Course Editor"),
        help=_('Enter the method by which this course is edited ("XML" or "Studio").'),
        default="Studio",
        scope=Scope.settings,
        deprecated=True  # Deprecated because someone would not edit this value within Studio.
    )
```
in the `common/lib/xmodule/course_module.py` file. Is the idea to create another attribute just like the object above? 

I'm kind of lost here, could you help me, giving another tip or something like this?

Thank you :)